tests/record-variant-tests.dx

```diff
@@ -84,39 +84,31 @@ Syntax for records, variants, and their types.
 
 :p
   x : {a:Int | b:Float} = {| a=3 |}
-  y : {a:Float | a:Int | ..._} = {| a | ...x |}
+  y : {a:Float | a:Int | ...} = {| a | ...x |}
   y
 > {|a| a = 3 |}
 
 
 'Parse errors
 
 :p {a:Int & b:Float | c:Int }
-
-> Parse error:94:21:
->    |
-> 94 | :p {a:Int & b:Float | c:Int }
->    |                     ^
-> unexpected '|'
-> expecting "..", "::", "<..", "=>", "{|", '$', '&', '.', '}', arrow, backquoted name, expression, infix operator, or resume
+> Syntax error: Bad field.  Expected `label : expr`,`... expr`, or a single `label`.
```

Comment:
I'm not sure if I understand this error message. Can we try to clarify it?

Reply:
Done.